<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_9</class>
 <widget class="QDialog" name="fct_comp_9">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion de réservations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Donner un login pour consulter les réservations correspondentes</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Login:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table_fct_comp_9">
     <column>
      <property name="text">
       <string>Nom</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prenom</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>noSpec</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>dateRep</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>noPlace</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>noRang</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_bas">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Choisir noSpec</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Choisir dateRep</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Choisir noRang</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Choisir noPlace</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QComboBox" name="box_ns_a"/>
       </item>
       <item>
        <widget class="QComboBox" name="box_dr_a"/>
       </item>
       <item>
        <widget class="QComboBox" name="box_nr_a"/>
       </item>
       <item>
        <widget class="QComboBox" name="box_np_a"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="push_ajout">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QComboBox" name="box_ns_s"/>
       </item>
       <item>
        <widget class="QComboBox" name="box_dr_s"/>
       </item>
       <item>
        <widget class="QComboBox" name="box_nr_s"/>
       </item>
       <item>
        <widget class="QComboBox" name="box_np_s"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="push_delete">
         <property name="text">
          <string>Suprrimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="push_fer">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_9</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_ns_a</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_9</receiver>
   <slot>refresh_dateRep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_dr_a</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_9</receiver>
   <slot>refresh_noRang()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_nr_a</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_9</receiver>
   <slot>refresh_noPlace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>push_ajout</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_9</receiver>
   <slot>add_reservation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>push_fer</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_9</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_ns_s</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_9</receiver>
   <slot>refresh_s_dateRep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_dr_s</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_9</receiver>
   <slot>refresh_s_noRang()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_nr_s</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>fct_comp_9</receiver>
   <slot>refresh_s_noPlace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>push_delete</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_9</receiver>
   <slot>delete_reservation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
  <slot>refresh_dateRep()</slot>
  <slot>refresh_noSpec()</slot>
  <slot>refresh_noRang()</slot>
  <slot>refresh_noPlace()</slot>
  <slot>add_reservation()</slot>
  <slot>refresh_s_dateRep()</slot>
  <slot>refresh_s_noRang()</slot>
  <slot>refresh_s_noPlace()</slot>
  <slot>delete_reservation()</slot>
 </slots>
</ui>
